### PR TITLE
Homogenize doc page titles/subheadings between the configs

### PR DESCRIPTION
**Pull request by rosteen**
These are the only inconsistencies I could find between the doc page titles in the different configs. I'm trying to be good and resist making more changes here 😆 .

### CODE REVIEW

File: docs/specviz/import_data.rst [hunk 1]
```diff
@@ -38,7 +38,7 @@ users can select which spectra that have be loaded to be visualized in the viewe
 
 .. _api-import:
 
-Loading data via the API
+Importing data via the API
 ------------------------
```
Comment (kecnry) [inline docs/specviz/import_data.rst:42]:
```suggestion
Importing data via the API
--------------------------
```

**Review by kecnry — COMMENTED** [review]
Hopefully fixing header underline lengths alone will fix the CI (and rebasing should fix the tests), then happy to approve/merge (seems quite uncontroversial).

**Comment by rosteen** [comment]
Good catch @kecnry, I went back and forth enough times on whether I prefer "Loading" or "Importing" that I missed those underline lengths.

**Review by pllim — APPROVED** [review]
LGTM. Thanks!

p.s. I slightly prefer "loading" but this is good enough. 😉

**Comment by pllim** [comment]
Unless you want a second approval, feel free to merge!

**Review by javerbukh — APPROVED** [review]

**Pull request merged**